Python/Image_Object_Detector/readme.md

```diff
@@ -0,0 +1,38 @@
+## LinkedIn Connectin Scraper
+
+A python script that scraps linkedin connections and saves it into a CSV file.
+
+## Tech Stack
+
+* Python
+	* Pandas
+	* Selenium
```

Comment:
Pandas and Selenium packages are not added in these project .

Reply:
oh, I actually take copies to not build things from scratch everytime, sometimes I miss these. xd
Ok 1 min will do them.